themes/bootstrap3/templates/search/versions.phtml

```diff
@@ -0,0 +1,116 @@
+<?php
```

Comment:
This seems like a lot of copy and paste from search/results.phtml. Can we instead simply include that template, and maybe use slots to adjust areas where differences are needed?

Reply:
I tried but found them to be sufficiently different that it would make the results template a bit of a mess. I still think they're better separate, but I moved the searchtools block to a sub-template to remove more of the copied code. What do you think?

Reply:
That certainly doesn't hurt, though I'm not sure it fully solves the problem. I think my biggest concern is that there's a lot of complex logic being duplicated here (for example, setting up the many variables used by the search box), and if that changes in the future, it's going to be very easy to forget to keep things in sync here. When I have more time, I'll try to do a closer comparison of the templates and see if I have any useful ideas.

Reply:
Please review commit 62afc1ff5cc4b47065ff6f8c2efef0f299548f61 -- I think this matches all of your custom template's functionality while eliminating all of the redundant logic, but it's possible I missed something.